layouts/_default/baseof.html

```diff
@@ -7,7 +7,12 @@
   {{ partial "head.html" . }}
 </head>
 <body class="">
-<div class="container">
+{{ if eq $.Site.Params.themeWidth "full" }}
+     <div class="container full">
+{{ else }}
+    <div class="container half">
+{{ end }}
```

Comment:
I would prefer this:

```
{{ $container := cond $.Site.Params.FullWidthTheme "container full" (cond $.Site.Params.CenterTheme "container center" "container") }}

<div class="{{- $container -}}">
```

I know it's soooooo verbose, but I think it's better than (error prone spaces):

```
{{ $themeSize := cond $.Site.Params.FullWidthTheme " full" (cond $.Site.Params.CenterTheme "  center" "") }}

<div class="container {{- $themeSize}}">
```